src/iop/clipping.c

```diff
@@ -1828,20 +1938,7 @@ void gui_update(struct dt_iop_module_t *self)
     keystone_type_populate(self, FALSE, p->k_type);
   }
 
-
-  /* special handling the combobox when current act is already selected
-     callback is not called, let do it our self then..
-   */
-  if(act == -1)
-  {
-    char str[128];
-    snprintf(str, sizeof(str), "%d:%d", abs(p->ratio_d), p->ratio_n);
-    dt_bauhaus_combobox_set_text(g->aspect_presets, str);
-  }
-  if(dt_bauhaus_combobox_get(g->aspect_presets) == act)
-    aspect_presets_changed(g->aspect_presets, self);
-  else
-    dt_bauhaus_combobox_set(g->aspect_presets, act);
```

Comment:
this is now gone in `aspect_presets_changed`, and the case where `dt_bauhaus_combobox_get(g->aspect_presets) == act` is now a no-op.